release_notes/371-release-notes.adoc

```diff
@@ -283,16 +283,24 @@ In the table, features are marked with the following statuses:
 [id="bug-fixes-371"]
 == Bug fixes
 
+[id="bug-fixes-371-2"]
+=== Resolved in version 3.71.2
+
+Release date: 13 October 2022
+
+* Before, because of a bug in {product-name-short} 3.71.1, the Collector pods stopped responding and reach a segmentation fault after allocating a memory block for the protocol buffer heap under certain load conditions.
```

Comment:
```suggestion
* Previously, because of a bug in {product-name-short} 3.71.1, the Collector pods stopped responding and reached a segmentation fault after allocating a memory block for the protocol buffer heap under certain load conditions.
```
There is nothing in our style information that prohibits using 'Before', I just thought 'Previously' sounded a bit better.